minard/views.py

```diff
@@ -467,17 +467,18 @@ def metric():
 
 @app.route('/eca')
 def eca():
+
     runs = ecadb.runs_after_run(0)      
     return render_template('eca.html', runs=runs)
  
-@app.route('/eca_run_detail/<int:run_number>')
```

Comment:
Yes, but render_template('eca_run_detail_%s.html' % run_type, run_number=run_number) appends the run number to a string and doesn't like an integer. I could change the template instead, but I decided rather to remove that conversion.